openshift/github-mirror-acceptance.yaml

```diff
@@ -64,6 +65,4 @@ parameters:
 - name: MEMORY_LIMIT
   value: 128Mi
 - name: CPU_REQUESTS
-  value: 300m
-- name: CPU_LIMIT
-  value: 300m
+  value: 100m
```

Comment:
why decreasing requests?

Reply:
because cpu usage is very low, decrease it to reflect real cpu requests, and make it easier to schedule